tests/functional/object_model/ovms_binary.py

```diff
@@ -82,6 +87,9 @@ def finalizer():
 
         return self.process, {"grpc_port": self.grpc_port, "rest_port": self.rest_port}
 
+    def stop(self):
```

Comment:
Why this method is empty?

Reply:
This flow isn't used in pre-commit testing.
It probably wasn't working for while and it requires checking.